### PR TITLE
Support annotations on enum member

### PR DESCRIPTION
**Pull request by LaylaLiu**

### CODE REVIEW

**Comment by dingeins** [comment]
Layla is a member of the OData team

File: test/FunctionalTests/Tests/DataEdmLib/FunctionalTests/CsdlParsingTests.cs [hunk 1]
```diff
@@ -2376,14 +2376,49 @@ public void TestEnumTypeParsingOutOfRange()
...
-            Assert.AreEqual(model.FindType("foo.Color"), badEnumMemeber.DeclaringType, "Bad member has correct type.");
-            Assert.IsTrue(badEnumMemeber.Value.IsBad(), "Bad member value is bad.");
-            Assert.AreEqual(EdmPrimitiveTypeKind.Int64, badEnumMemeber.Value.Type.PrimitiveKind(), "Bad members bad value has correct type.");
+            IEdmEnumMember enumMemeber = ((IEdmEnumType)model.FindType("foo.Color")).Members.First(m => m.Name == "Green");
+            Assert.AreEqual(model.FindType("foo.Color"), enumMemeber.DeclaringType, "Enum member has correct type.");
+            Assert.IsTrue(enumMemeber.Value.IsBad(), "Enum member value is bad.");
+            Assert.AreEqual(EdmPrimitiveTypeKind.Int64, enumMemeber.Value.Type.PrimitiveKind(), "Enum member bad value has correct type.");
+        }
+
+        [TestMethod]
```
Comment (congysu) [inline test/FunctionalTests/Tests/DataEdmLib/FunctionalTests/CsdlParsingTests.cs]: Not for your change.  But if we will create a new test class, consider using xUnit.

**Comment by MrTomWhite** [comment]
Why did this PR get closed?

**Comment by piotrpMSFT** [comment]
@LaylaLiu, did you intend to delete the branch before the PR was merged? It would be great to see this change make it into the library!

**Comment by LaylaLiu** [comment]
@MrTomWhite  I have merged this pull request into the odata.net master.

**Comment by LaylaLiu** [comment]
@piotrpMSFT, The change in this pull request has been checked into the master branch.

**Pull request closed without merging**